subprojects/gvdb.wrap

```diff
@@ -0,0 +1,2 @@
+[wrap-redirect]
+filename = glib-2.74.4/subprojects/gvdb.wrap
```

Comment:
Whoops, this was unintentional (looks like a byproduct of sanity checker script).